Description: Work around zsh completion generation breakage
 which seems to be introduced by clap_complete. Should be resolved
 after that is updated.
Forwarded: not-need
Last-Update: 2024-06-14
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/completions.rs
+++ b/src/completions.rs
@@ -139,3 +139,3 @@
   (
-    r#"    _arguments "${_arguments_options[@]}" : \"#,
+    r#"    _arguments "${_arguments_options[@]}" \"#,
     r"    local common=(",
